samples/serialport-arduino/arduino-demo.csproj

```diff
@@ -4,12 +4,12 @@
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp3.0</TargetFramework>
     <RootNamespace>arduino_demo</RootNamespace>
-    <!-- this can also be passed from the command line (i.e. dotnet publish -r linux-arm) -->
     <RuntimeIdentifier>linux-arm</RuntimeIdentifier>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="System.IO.Ports" Version="$(SystemIOPortsPackageVersion)" />
+    <PackageReference Include="System.Device.Gpio" Version="1.1.0-prerelease.20153.1" />

```

Comment:
Just as a comment we will have to come up with a way of updating all of these versions on all of our packages every time we ship a new NuGet.

Reply:
Yes. We should be doing that anyway.